nat-lab/tests/uniffi/serialization.py

```diff
@@ -0,0 +1,62 @@
+import inspect
+import Pyro5.api  # type: ignore
+
+
+def init_serialization(libtelio):
```

Comment:
Hmm, how is this serialization used?

Reply:
I added a comment to the function to explain a but, but basically it's called at the start of  `libtelio_proxy.py` and `libtelio_remote.py` which makes Pyro5 able to (de)serialize our types on both the test host and in the test containers/VMs